create-neon/src/bin/create-neon.ts

```diff
@@ -29,31 +30,60 @@ function inferVersions(): Versions {
   };
 }
 
+function deleteNeonDir(dir:string): Promise<void> {
+  return fs.rm(dir, { force: true, recursive: true });
+}
+
 async function main(name: string) {
+  let workingDirPath: string = process.cwd()
+  let tmpDirPath: string = os.tmpdir()
+  let tmpFolderName:string|undefined;
+
   try {
-    await fs.mkdir(name);
+    //check if folder or file with same name exists
+    await fs.mkdir(name)
+    await fs.rmdir(name)
+
+    //change current working directory to /temp
+    process.chdir(path.join(tmpDirPath))
```

Comment:
`path.join` is unnecessary here. However, my preference is to use an absolute path for this directory instead of relying on the current working directory.

Can you do something like this instead?

```js
let tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), `${name}-`));
```